.github/workflows/wkdev-sdk.yml

```diff
@@ -10,14 +10,12 @@ defaults:
   run:
     shell: bash
 jobs:
-  build:
-    runs-on: self-hosted
+  build_amd64:
+    runs-on: [self-hosted==amd64]
```

Comment:
That's neat, I wasn't aware that it's trivial to select the arch this way.

Reply:
No, I checked the docs. The original (self-hosted, x64) was correct